README.md

```diff
@@ -1,146 +1,219 @@
 # witnet-ethereum-bridge [![](https://travis-ci.com/witnet/witnet-ethereum-bridge.svg?branch=master)](https://travis-ci.com/witnet/witnet-ethereum-bridge)
 
-`witnet-ethereum-bridge` is an open source implementation of a bridge
-from Ethereum to Witnet. This repository provides several contracts:
+`witnet/witnet-ethereum-bridge` is an open source implementation of an API that enable Solidity smart contract developers to harness the full power of the [**Witnet Decentralized Oracle Network**](https://docs.witnet.io/overview/concepts/).
```

Comment:
```suggestion
`witnet/witnet-ethereum-bridge` is an open source implementation of an API that enables Solidity smart contract developers to harness the full power of the [**Witnet Decentralized Oracle Network**](https://docs.witnet.io/overview/concepts/).
```

"the full power"
![image](https://user-images.githubusercontent.com/12464544/128492915-08289218-7df7-4e8e-aa98-3aa6f79e2cc8.png)